azurerm/internal/services/loganalytics/log_analytics_linked_storage_account_resource.go

```diff
@@ -47,7 +48,6 @@ func resourceArmLogAnalyticsLinkedStorageAccount() *schema.Resource {
 					strings.ToLower(string(operationalinsights.CustomLogs)),
 					strings.ToLower(string(operationalinsights.AzureWatson)),
 					strings.ToLower(string(operationalinsights.Query)),
-					strings.ToLower(string(operationalinsights.Ingestion)),
```

Comment:
this'd be a breaking change - is this still supported?

Reply:
It's not, unfortunately:
```
// LinkedStorageAccountsProperties linked storage accounts properties.
type LinkedStorageAccountsProperties struct {
	// DataSourceType - READ-ONLY; Linked storage accounts type. Possible values include: 'CustomLogs', 'AzureWatson', 'Query', 'Alerts'
	DataSourceType DataSourceType `json:"dataSourceType,omitempty"`
	// StorageAccountIds - Linked storage accounts resources ids.
	StorageAccountIds *[]string `json:"storageAccountIds,omitempty"`
}
```

Reply:
```
// DataSourceType enumerates the values for data source type.
type DataSourceType string

const (
	// Alerts ...
	Alerts DataSourceType = "Alerts"
	// AzureWatson ...
	AzureWatson DataSourceType = "AzureWatson"
	// CustomLogs ...
	CustomLogs DataSourceType = "CustomLogs"
	// Query ...
	Query DataSourceType = "Query"
)
```

Reply:
Often they remove valid values from the SDK, but they can still be sent to the API and the only way to know one way or another is by sending it off in a test 🙁

Reply:
I'll test after a meeting here.

Reply:
@katbyte you were right, it still just works. Restoring this one.

Reply:
of course it does hah, azure being azure - thanks!